tests/foreman/ui/test_host.py

```diff
@@ -377,6 +374,14 @@ def test_positive_create_libvirt(self):
             )
             name = u'{0}.{1}'.format(hostname, self.domain_name)
             self.assertIsNotNone(self.hosts.search(name))
+            for _ in range(25):
+                result = self.hosts.get_host_properties(name, ['Build'])
+                if result['Build'] == 'Pending installation':
+                    sleep(30)
+                else:
+                    break
+            self.assertEqual(result['Build'], 'Installed')
+            self.hosts.delete(name, dropdown_present=True)
```

Comment:
Cleanup func or try..finally maybe? Otherwise unsuccessful installation won't be deleted.

Reply:
we have delete in separate test case and I was thought to not put delete here at all, but we have to make that cleanup.
Okay, if `self.assertIsNotNone(self.hosts.search(name))` is positive, I will wrap up code in try, finally

Reply:
think a cleanup function will be a better choice